toolbox

```diff
@@ -18,7 +18,7 @@
 
 exec 3>/dev/null
 
-arguments="$@"
+arguments="$*"
```

Comment:
I tweaked your commit to use the trick from [Rich's sh tricks](http://www.etalabs.net/sh_tricks.html) in #86. I hope I got it right. I also updated the commit message to explain the change in more detail because of the arcane nature of the shell concepts involved. :)

Comments welcome.

Reply:
I merged #86 after a few hours of testing. Seems to work as intended.